Formula/dmd.rb

```diff
@@ -81,13 +81,23 @@ def install
       system "make", "install", *make_args
     end
 
-    bin.install "generated/osx/release/64/dmd"
+    on_macos do
+      bin.install "generated/osx/release/64/dmd"
+    end
+    on_linux do
+      bin.install "generated/linux/release/64/dmd"
+    end
     pkgshare.install "samples"
     man.install Dir["docs/man/*"]
 
     (include/"dlang/dmd").install Dir["druntime/import/*"]
     cp_r ["phobos/std", "phobos/etc"], include/"dlang/dmd"
-    lib.install Dir["druntime/lib/*", "phobos/**/libphobos2.a"]
```

Comment:
Wondering if we could install everything on Mac too. Can you check what other files are there on Mac? Installing shared libraries on Mac should be fine too.

Reply:
The only libraries on Mac are `libdruntime.a` and `libphobos2.a`

Also, the current Mac bottles are missing `libdruntime.a`